ROLES.md

```diff
@@ -0,0 +1,58 @@
+* In TSC, Maintainers have equal voting rights, and there is no other hierarchy within this group
+* Effectively they are the product managers and core developers of the Design System, leading the RFC process and facilitating working groups in order to make consensus decisions over technical architecture and future direction
+* The TSC are responsible managing contributions to the Design System made by supporting organisations, Government agencies and individuals in the broader community
+* Maintainers will be attributed [location TBD]
+
+## Supporting Organisations
+
+> Supporting Organisations describes any non-government entity.
+
+* Supporting Organisations are third parties who play an active and ongoing role in supporting the work of the TSC and the Design System itself
+* These organisations may include non-for-profits and NGOs, commercial vendors and Government organisations (operating in a different context to use of the Design System in delivery which is covered below) with a vested interest in the sustainability of the Design System
+* Support could take multiple forms, including but not limited to:
+    * Resourcing - their employees provide hands-on technical or non-technical assistance to the TSC
+    * Advice - their employees provide guidance to the TSC on specific issues
+    * Promotion - helping drive awareness of the Design System and performing community outreach
+    * Financial - sponsorship or direct funding to cover costs incurred in managing and supporting the Design System, running community events
+* Their engagement with the Design System will be formally defined by a Partnership Agreement [TBD] and will be published in the open
+* Supporting Organisations will be attributed on the website [location TBD]
+
+## Government Organisations
+
+> The word "Government Organisations" is a collective noun that describes all government entities, including agencies, departments, offices and so on.
+
+* The primary way Government organisations will engage with the Design System is through using it in the delivery of their digital products and services
+* Government organisations have expectations of the Design System (suitability for its purpose, ongoing development, clarity on future features) that must be met by the community for them to be comfortable using it
+* The TSC expect Government organisations to act in the best interest of the Design System, which includes following the [MIT License](LICENSE), using our provided solutions for reporting issues and requesting changes, and sharing their work and findings with the community
+* Beyond their own delivery work, Government organisations may also act in support of the Design System & TSC, as defined above.
+* Supporting Government Organisations will be attributed on the website [location TBD]
+
+## Contributors (on behalf of Government Organisations)
```

Comment:
@sawks I added this role definition

Reply:
cool - it might be redundant at the moment as the responsibilities overlap with individual contributors, but happy to leave it in to provide structure if either of those role definitions change in the future...